db/migrate/20230403112350_add_details_to_products.rb

```diff
@@ -0,0 +1,7 @@
+class AddDetailsToProducts < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
this will fire 3 queries. is there a way we can do that in 1 query?

Reply:
change_table :products do |t|
      t.column :enabled, :boolean
      t.column :discount_price, :decimal, precision: 8, scale: 2
      t.column :permalink, :string
    end